src/App.tsx

```diff
@@ -291,17 +292,22 @@ function AppContent() {
         );
       }
 
-      snackbar.pageInfo(message, strings.TIME_ZONE_INITIALIZED_TITLE, {
-        label: strings.GOT_IT,
-        apply: () => {
-          if (notifyUser) {
-            updatePreferences('timeZoneAcknowledgedOnMs', Date.now());
-          }
-          if (notifyOrg) {
-            updatePreferences('timeZoneAcknowledgedOnMs', Date.now(), selectedOrganization?.id);
-          }
+      snackbar.pageInfo(
+        message,
+        strings.TIME_ZONE_INITIALIZED_TITLE,
+        {
+          label: strings.GOT_IT,
+          apply: () => {
+            if (notifyUser) {
+              updatePreferences('timeZoneAcknowledgedOnMs', Date.now());
+            }
+            if (notifyOrg) {
+              updatePreferences('timeZoneAcknowledgedOnMs', Date.now(), selectedOrganization?.id);
+            }
+          },
         },
-      });
+        'org'
```

Comment:
@constanzauanini you would need to set this as 'user' for weight unit notification